clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -955,11 +957,26 @@ class ArchiveFileHandler final : public FileHandler {
   StringMap<unsigned>::iterator CurrBundle = Bundles.end();
   StringMap<unsigned>::iterator NextBundle = Bundles.end();
 
+  /// Output mode for the archive unbundler.
+  enum class OutputType {
+    Unknown,
+    FileList, // Output is a list file with extracted object file names
+    Object,   // Output is a single object file
+    Archive   // Output is an archive wtih extracted objects
```

Comment:
Can we replace `FileList` with `Archive`?
I'm not suggesting doing it right away, but I don't understand the difference between these types, which can justify having both of them. Considering that `Archive` is already supporting in llorg, it seems reasonable to replace `FileList` with `Archive`. What do you think?

Reply:
I'm sure once we change from `aoo` to `a` usage in the driver, we can get rid of the FileList type.

Reply:
Yes, that is the plan.